### ISSUE TITLE
Camera3D and Viewport functions for mask culling use layers 1-20 and 0-19

### ISSUE THREAD

**Issue by Cykyrios** [godotengine/godot#76608]
### Godot version

v4.0.2.stable.official [7a0977ce2]

### System information

Linux Manjaro, kernel 6.1.25, X11, KDE Plasma 5.27.4

### Issue description

Layers are numbered 1-20 everywhere in the editor. However, there is a discrepancy in the functions used to edit the visibility of those layers in Camera3D and Viewport functions:
* `Camera3D.set_cull_mask_value()` expects a layer number in the 1-20 range
* `Viewport.set_canvas_cull_mask_bit()` expects a number in the 0-19 range

3D and canvas layers may not be directly related, but I think it is strange that both do not use the same numbering.

### Steps to reproduce

* Create a scene containing both a SubViewport and a Camera3D.
* Disable a layer in the camera using `Camera3D.set_cull_mask_value(layer, false)`
* Do the same in the subviewport with `Viewport.set_canvas_cull_mask_bit(layer, false)`
* Print `Camera3D.cull_mask` and `Viewport.canvas_cull_mask`, the values will be different.

### Minimal reproduction project

[MRP.zip](https://github.com/godotengine/godot/files/11361019/MRP.zip)

In this example, I pass 5 as the argument for both the viewport and camera, but the viewport actually disables layer 6.
```gdscript
set_canvas_cull_mask_bit(5, false)
camera.set_cull_mask_value(5, false)
print("Viewport cull mask: %d" % [canvas_cull_mask])
print("Camera3D cull mask: %d" % [camera.cull_mask])
```
The output is the following:
```
Viewport cull mask: 4294967263
Camera3D cull mask: 1048559
```

**Comment by viksl** [comment]
Someone might correct me if it's different in this case but even docs talks about this in this sense so, bit and value are not the same things. When you hover over mask or layers and such in godot you will see an associated value with a bit.
One of your methods sets bits `_bit` and the other `_value`.

I don't think this is necessarily a bug since even relevant methods in docs talk about bit/value.
Although I do understand the confusion and perhaps this could serve more of a proposal for improvement rather than a bug report, in 4.0 I believe couple methods to work with values rather than bits were added to make it easier to work with but it seems here and there the engine still needs some extra touches.

Feel free to correct me if I'm wrong. ;-)

**Comment by adamscott** [comment]
Maybe we could add utility functions like `Camera3D.set_cull_mask_bit()` and `Viewport.set_canvas_cull_mask_value()`?

**Comment by viksl** [comment]
Collision objects and navigation already have bit/value so it might be worth adding the same behvaiour to where only bit option is at this point, quick search through docs seems it's just few cases, or maybe just one? I might have overlooked something.

**Comment by Cykyrios** [comment]
I agree it's not a bug per se, just feels like somewhat of a usability issue. It could indeed help to have functions for both bits and values, or make it clear in the documentation that the difference exists.
